src/Tabs.js

```diff
@@ -21,25 +25,29 @@ class Tabs extends Component {
     if (onChange) onChange(idx, e);
   }
 
-  componentWillReceiveProps(nextProps) {
-    const { tabOptions = {} } = nextProps;
+  componentDidUpdate() {
```

Comment:
any reason for changing this lifecycle?

Reply:
Yes, it doesn't work correctly otherwise.

If you want to control active tab programmatically by passing 'active' property - $().tabs() should be called after it's rendered, while componentWillReceivedProps() will call it before - the tabs are out of sync after that.

Plus React are advising to use componentDidUpdate() and not use componentWillReceiveProps() anyway, so it seems like a good idea to change it.